packages/mongodb-chatbot-ui/package.json

```diff
@@ -36,7 +36,7 @@
     }
   },
   "scripts": {
-    "build": "if [[ \"$ENVIRONMENT\" == \"qa\" ]]; then npm run build:static-site:qa && echo \"qa\"; else npm run build:static-site:staging; fi",
+    "build": "npm run build:component && if [[ \"$ENVIRONMENT\" == \"qa\" ]]; then npm run build:static-site:qa && echo \"qa\"; else npm run build:static-site:staging; fi",
```

Comment:
to build component from `npm run build` which is used in drone to build the component package